provider/oracle/instance_test.go

```diff
@@ -59,13 +60,11 @@ func (i instanceSuite) TestNewOracleInstance(c *gc.C) {
 }
 
 func (i instanceSuite) TestId(c *gc.C) {
-	instance, err := oracle.NewOracleInstance(oracletesting.DefaultFakeInstancer.Instance, i.env)
+	inst, err := oracle.NewOracleInstance(response.Instance{Name: "/Compute-a432100/sgiulitti@cloudbase.com/juju-f84133-0/ebc4ce91-56bb-4120-ba78-13762597f837"}, i.env)
```

Comment:
Why can't we use the oracletesting.DefaultFakeInstancer.Instance? If the Instance.Name is not as it should be, we can just change that?
 Would prefer not to hard code another looooong string here when it's already been set up elsewhere.

Reply:
Instance.Name changes for every new OracleEnviron, yet the DefaultFakeInstance is hardcoded.  Changed back to using oracletesting.DefaultFakeInstancer.Instance here.